src/shared/src/components/LandmarkShortcuts.tsx

```diff
@@ -0,0 +1,65 @@
+import { makeStyles } from "@material-ui/core";
+import React from "react";
+import cn from 'classnames';
+
+export interface ILandmarkShortcutsProps {
+  navToMainText: string;
+}
+
+const useStyles = makeStyles({
+  button: {
+    textDecoration: 'underline',
+    textAlign: 'left',
+    width: 'fit-content',
+  },
+  jumpNav: {
+    backgroundColor: "#fff",
+  },
+  'sr-only': {
```

Comment:
maybe move the content of this class into sr-only-focusable

Reply:
Done ✔️